docs/source/optim.rst

```diff
@@ -113,6 +113,11 @@ Algorithms
     :members:
 .. autoclass:: AdamW
     :members:
+.. autoclass:: AdamW2
```

Comment:
What is AdamW2?

There is an implementation of AdamW in PyTorch since #21250. In the version implemented, the decay is taken from theta, but the weight and lr schedulers are still coupled. I would expect the new implementation to supersede the current one.

Reply:
Sorry, AdamW2 was a relict from comparing the old with the new optimizer. 
I've now copied the same update step to the wrapper.